src/OneExplorer/OneExplorer.ts

```diff
@@ -1188,3 +1243,10 @@ input_path=${modelName}.${extName}
     return this._tree;
   }
 }
+
+type CfgInfo = {
```

Comment:
How about moving this type definition nearby the related function or to the top?

Reply:
I move this type to the top of this file :-D